src/GameDataReader/Common/Refractor/V1/Files/UtilityMethodsRefractorV1ConfigFile.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace GameDataReader.Common.Refractor.V1.Files;
+
+internal class UtilityMethodsRefractorV1ConfigFile
+{
+    public string GetProcessPath(string modName)
+    {
+        var matchingProcesses = Process.GetProcessesByName(modName);
+        return matchingProcesses.Length == 1 ? matchingProcesses[0].GetMainModuleFileName(modName) : string.Empty;
+    }
+
+    public string GetConFilePath(string conFile, string gameProcessPath, string gameName)
+    {
+        var appData = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+        var executingPath = File.Exists($@"{gameProcessPath}{conFile}") ? $@"{gameProcessPath}{conFile}" : string.Empty;
+        var fallbackPath = File.Exists($@"{appData}\VirtualStore\Program Files (x86)\EA GAMES\{gameName}{conFile}") ? $@"{appData}\VirtualStore\Program Files (x86)\EA GAMES\{gameName}{conFile}" : string.Empty;
+        return File.Exists($@"{gameProcessPath}{conFile}") ? executingPath : fallbackPath;
```

Comment:
This seems a bit chaotic. You are building all paths multiple times and are even checking twice if `$@"{gameProcessPath}{conFile}"` exists. This could much cleaner with regular if statements and early returns, e.g.

```c#
var executingPath = $@"{gameProcessPath}{conFile}";
if (File.Exists(executingPath)) {
    return executingPath;
}
...
return string.Empty;
```

That was you get rid of the ternary operators, only have build each path once and only check if the path exists. Plus you don't do any additional I/O operations once you already know you'd be returning `executingPath`.

And on a more general note: I am a big proponent of using path libraries to join paths (or manipulate paths in any other way). You'll usually end up with much more robust code than when simply concatenating strings. I know GameDataReader currently simply uses string concatenation everywhere, so that is something to maybe change in a later refactor.